brokenspoke_analyzer/cli/root.py

```diff
@@ -55,8 +59,43 @@ def callback(verbose: int = typer.Option(0, "--verbose", "-v", count=True)) -> N
     logger.add(sys.stdout, format=log_format, level=log_level, colorize=True)
 
 
+def _version_callback(value: bool) -> None:
+    # Get the package's version
+
+    package_version = metadata.version("brokenspoke-analyzer")
+    if value:
+        print(f"brokenspoke-analyzer version: {package_version}")
+        raise typer.Exit()
+
+
 # Create the CLI app.
 app = typer.Typer()
+
+
+@app.callback()
+def callback(
+    version: Annotated[
+        Optional[bool],
+        typer.Option(
+            "--version",
```

Comment:
Can we also add the short version ('-v') and the count option? This way we could specify `bna -v` for lite logging or `bna -vvv` very verbose logging.

Reply:
No problem! I added the short version to the verbosity Option.